Re-adding science page for percusion (#367)
orcestra-campaign/book: orcestra_book/science.md

# Science

## The scientific motivation for PERCUSION
![](./figures/ITCZ_ferrel.jpg)
*Figure 1. William Ferrel's schematic depiction of Earth’s surface wind. Figure taken from Birner et al, 2014 and adapted from Ferrel, 1856.*

Most tropical rain is concentrated within a narrow latitudinal band known as the Inter-Tropical Convergence Zone (ITCZ). This band shifts in position, width and longitude on timescales from days to millenia, with consequences for the billions of people who live within its reach.

These variations in the ITCZ have been linked to changes in the atmosphere's energy balance, but this balance is itself hard to constrain --- it depends on mesoscale circulations which are notoriously difficult to measure (vertical winds are usually an order of magnitude smaller than horizontal winds). Without these observations, we are unable to constrain our theories and computer simulations, limiting our ability to predict future changes in the width and position of the ITCZ.

> Will the islands of Cape Verde, which lie at the edge of the ITCZ, get wetter or dryer in the next 50 years? What about about the Sahel? We cannot yet confidently say what will happen.

Reviewer: ```suggestion
Will the islands of Cape Verde, which lie at the edge of the ITCZ, get wetter or dryer in the next 50 years? What about the Sahel? We cannot yet confidently say what will happen.
```

> To break this deadlock, one of PERCUSION's scientific objectives is to measure the mesoscale vertical velocity $w$ at the center and edges of the ITCZ. Variability in $w$ will be characterized over 20+ flights between August and October in the East and West Atlantic. This will allow, for the first time, a process-level observational estimate of the ITCZ's circulation and energy export. With this information, we will be able to peer into the inner life of the ITCZ and understand why it billows and blows.

Reviewer: ```suggestion
To break this deadlock, one of PERCUSION's scientific objectives is to measure the mesoscale vertical velocity $w$ at the center and edges of the ITCZ. Variability in $w$ will be characterized over 20+ flights in August and September in the East and West Atlantic. This will allow, for the first time, a process-level observational estimate of the ITCZ's circulation and energy export. With this information, we will be able to peer into the inner life of the ITCZ and understand why it billows and blows.
```

```{admonition} Knowledge Check
:class: tip dropdown

> Want to learn more? Then you are in the right place. Our goal here is to provide a minimal introduction to the science that motivates PERCUSION.

Reviewer: ```suggestion
Want to learn more? Then you are in the right place. Our goal here is to provide a minimal introduction to the science that motivates [PERCUSION](percusion.md).
```

From this constraint, one can deduce the location of the ITCZ by looking at where the vertically integrated meridional energy transport of moist static energy ($\partial_y \langle v m \rangle$) changes direction (Kang et al, 2008; Schneider et al, 2014). Determining the change in width of the ITCZ requires additional considerations about the mass balance of the Hadley Cell (Byrne and Schneider, 2016), and it primarily depends on the Gross Moist Stability ($\mathcal{M}$) of the atmosphere, a measure of how efficient the export of moist static energy from the ITCZ is.

Thus, if one can understand where moist static energy is being exported ($\partial_y \langle v m \rangle$) and how efficiently this is done ($\mathcal{M}$), then one can obtain a basic understanding of where the ITCZ lies and how wide it should be. 

```{admonition} Caution
:class: warning dropdown

> The trouble is, as will soon see, that these theories are hard to constrain theoretically and observationally, which limits our ability to predict future changes in rainfall. Relevant questions such as how the width of the ITCZ change as a function of longitude also remain unclear. This calls for the need to improve our observations of moist static energy export and its efficiency.

Reviewer: ```suggestion
The trouble is, as we will soon see, that these theories are hard to constrain theoretically and observationally, which limits our ability to predict future changes in rainfall. Relevant questions such as how the width of the ITCZ change as a function of longitude also remain unclear. This calls for the need to improve our observations of moist static energy export and its efficiency.
```

```{admonition} Knowledge Check
:class: tip dropdown

**To summarize:** we know that Gross Moist Stability $\mathcal{M}$ is a key quantity in predicting the ITCZ's changes, but we don't understand it well because it is hidden within the ITCZ's inner life, that is, its dynamic and thermodynamic mesoscale structure.
```

### The role of PERCUSION
To overcome this challenge, PERCUSION seeks to measure the mesoscale circulations within and at the edges of the ITCZ. The key hypothesis is that measuring these circulations and their occurrence frequency, along with calculating the energy export, will advance our understanding of the changes in width and location of the ITCZ and improve our predictions of future rainfall.

> To do this, PERCUSSION follows the successful application of dropsonde circles during the EUREC4A campaign to infer mesoscale circulations (Bony, 2017). Dropsonde circles of about 130 km in radius will be placed at the center and the edges of the ITCZ on each of the 20+ flights planned for August - October 2024. The dropsondes measure the horizontal wind as a function of height (via a gps tracking the drift of the sonde), and then the mesocale vertical wind of the entire circle can be inferred from the divergence theorem.

Reviewer: ```suggestion
To do this, PERCUSION follows the successful application of dropsonde circles during the EUREC4A campaign to infer mesoscale circulations (Bony, 2017). Dropsonde circles of about 130 km in radius will be placed at the center and the edges of the ITCZ on each of the 20+ flights planned for August - October 2024. The dropsondes measure the horizontal wind as a function of height (via a gps tracking the drift of the sonde), and then the mesocale vertical wind of the entire circle can be inferred from the divergence theorem.
```

![](./figures/doldrums_hypotheses.jpg)
*Figure 7. Potential causes for the low-wind speed events known as the "doldrums". Credit: Julia Windmiller.*

> In addition to pursuing a better understanding of the ITCZ, PERCUSSION hopes to use these budgets of moist static energy and observed mesoscale circulations to test hypothesized controls on convective aggregation such as the up-gradient horizontal transfer of moist static energy (Muller, 2022); and to test hypothesized circulation controls on the doldrums such as the association between subsiding air and surface wind divergence (Figure 7 and Windmiller, 2024).

Reviewer: ```suggestion
In addition to pursuing a better understanding of the ITCZ, PERCUSION hopes to use these budgets of moist static energy and observed mesoscale circulations to test hypothesized controls on convective aggregation such as the up-gradient horizontal transfer of moist static energy (Muller, 2022); and to test hypothesized circulation controls on the doldrums such as the association between subsiding air and surface wind divergence (Figure 7 and Windmiller, 2024).
```

> # Science

Author: I have moved what was on `science.md` to` percusion.md` as a focus area and removed science.md from `_toc.yml`.

Will the islands of Cape Verde, which lie at the edge of the ITCZ, get wetter or dryer in the next 50 years? What about the Sahel? We cannot yet confidently say what will happen.

To break this deadlock, one of PERCUSION's scientific objectives is to measure the mesoscale vertical velocity $w$ at the center and edges of the ITCZ. Variability in $w$ will be characterized over 20+ flights in August and September in the East and West Atlantic. Each of these flights will include an overpass from the new EarthCare satellite, thus allowing us to connect process-level observations with longer term measurements. When these dynamics are connected with quantties such as vorticity, divergence, cloud condensates, surface fluxes and radiative heating, this will allow, for the first time, a process-level observational estimate of the ITCZ's circulation and energy export. With this information, we will be able to peer into the inner life of the ITCZ and understand why it billows and blows.

> ```{admonition} Knowledge Check

Author: Agreed.

where $z_s$ is the surface and $z_\text{TOA}$ is the top of the atmosphere. (One can show this proportionality by starting with the definition of Gross Moist Stability from Equation 2.3 of Raymond et al, 2009, then assuming horizontal gradients in moist static energy to be zero, and then integrating by parts. The boundary term should be zero since $w$ is zero at the surface and the top of the atmosphere.) 

The benefit of this re-arrangement is that it makes clear the connection between Gross Moist Stability $\mathcal{M}$ and horizontal convergence/divergence ($\partial_z w$).

![](./figures/mse_profiles.jpg)
*Figure 6. Typical moist static energy profile of the tropical atmosphere (here denoted by $h$ rather than $m$) and the dependence of moist static energy export on the type of circulation. The vertical line represents the profile of a convecting air parcel. Figure taken from Julia Windmiller (see also Raymond et al, 2009).*

At the edges of the ITCZ, we generally expect an import of moist static energy at the surface due to the convergence of warm and moist trade winds. The export of this energy aloft, however, will depend on the type of mesoscale circulation present. 

> For a typical c-shaped moist static energy profile of the environment (Figure 6), $\mathcal{M}$ will depend on the outflow level of the circulation. A shallow circulation will export a smaller amount of $m$ than a deep circulation will, since $m$ is less in the mid troposphere than the upper troposphere. Additionally, the nonlinearity of $m$ with height suggests that $\mathcal{M}$ computed with a zonal-mean profile of the ITCZ circulation may not be the same as computing $\mathcal{M}$ at the mesoscale and *then* taking the zonal mean.

Author: Agreed that we can avoid such details here.

To do this, PERCUSION follows the successful application of dropsonde circles during the EUREC4A campaign to infer mesoscale circulations (Bony, 2017). Dropsonde circles of about 130 km in radius will be placed at the center and the edges of the ITCZ on each of the 20+ flights planned for August - September 2024. The dropsondes measure the horizontal wind as a function of height (via a gps tracking the drift of the sonde), and then the mesocale vertical wind of the entire circle can be inferred from the divergence theorem.

We need to be able to identify the edges and the center of the ITCZ. Ideally, we should use a metric that is forecastable yet physically relevant. Although we will sometimes look directly at surface winds, clouds, and precipitation to identify the edges of the ITCZ; we choose to primarily look at precipitable water since it is bimodal --- its sharp geographic gradients separate the dry and moist regions of the tropics and often coincide with the convergent winds, clouds and precipitation associated with the ITCZ (Mapes 2018, Masunaga and Mapes 2020). 

> This bimodality is intriguing, since it is a key feature of self aggregation, an emergent property of clouds-circulation coupling which is seen in idealized numerical models (Wing et al, 2018). What role do the processes that sharpen the gradients in dry and moist regions in idealized simulations play in reality? If they do play a role, are they temperature dependent like in idealized simulations? This would have implications for future rainfall and for radiative feedbacks since the areal ratio of moist to dry tropics is a regulator of Earth's climate stability (Pierrehumbert, 1995).

Author: Would it be worth adding the focus area on Self Aggregation right now, but then indicating that it is a "stub article"? Short wikipedia articles often do this, and then they invite the reader to make changes.  The same suggestion goes for the paragraph below on why HALO meets with EarthCare.

I'm also okay just getting rid of these paragraphs, especially if others would rather start afresh when they add a new focus area.

> This bimodality is intriguing, since it is a key feature of self aggregation, an emergent property of clouds-circulation coupling which is seen in idealized numerical models (Wing et al, 2018). What role do the processes that sharpen the gradients in dry and moist regions in idealized simulations play in reality? If they do play a role, are they temperature dependent like in idealized simulations? This would have implications for future rainfall and for radiative feedbacks since the areal ratio of moist to dry tropics is a regulator of Earth's climate stability (Pierrehumbert, 1995).

Author: I've now removed this paragraph since it's easy enough to bring it back in the future if necessary.

> We tend to use the steepest gradient in precipitable water to identify the edges of the ITCZ to allow for statistical sampling, but assesing how mechanisms change over time requires a longer measurement period. To extend these measurements in space and time, a portion of each flight will coincide with an overpass of the new EarthCare "cloud observatory" satellite. The aircraft we fly on, HALO, was designed to carry a similar payload. By flying together, we help to validate EarthCare, which can provide us with greater statistical insights into the relationships we observe while flying, and additional measurements such as top of atmosphere energy balance and profiles of atmospheric radiative heating. This validation also enhances the confidence in EarthCare itself and will benefit the climate research community.

Author: I've now removed this paragraph since it's easy enough to bring it back in the future if necessary.
